rules/CVE-2021-25742/raw.rego

```diff
@@ -6,7 +6,15 @@ deny[msga] {
 	image := deployment.spec.template.spec.containers[i].image
```

Comment:
For the future support it would be nice to have an example of the image string here.